node/lib/openshift-origin-node/model/application_container_ext/environment.rb

```diff
@@ -273,13 +272,18 @@ def validate_ssh_keys(ssh_keys)
 
         # Add user environment variable(s)
         def user_var_add(variables, gears = [])
+          config = ::OpenShift::Config.new
```

Comment:
Why? Other modules under ApplicationContainerExt that use `OpenShift::Config` do not `require 'openshift-origin-common'`. I believe this is because `../application_container.rb` contains `require 'openshift-origin-common'`